src/librustc_typeck/diagnostics.rs

```diff
@@ -3645,55 +3645,63 @@ fn main() {
 "##,
 
 E0520: r##"
-A non-default implementation was already made on this type
-implementation so it cannot be specialized afterward. Erroneous
-code example:
+A non-default implementation was already made on this type so it cannot be
+specialized further. Erroneous code example:
 
 ```compile_fail
 #![feature(specialization)]
 
-trait SpaceLama {
+trait SpaceLlama {
```

Comment:
@bluss: Well, I invite you to take a look at my previous error explanations/test cases PRs then. :p